README.md

```diff
@@ -55,6 +55,8 @@ module "vote_service_sg" {
 }
 ```
 
+**Note:** it is not possible to use variable outputs from this module or other modules that contain calculated values when defining the security group resources. This is typically an issue when specifying either `ingress_with_source_security_group_id` or `egress_with_source_security_group_id` parameters and attempting to use the security group id of a resource which has not yet been created. However referencing variables that are already "hard-coded" in the .tf file (i.e. not calculated values dependent on the infrastructure being created) are fine. E.g. the VPC cidr block `"10.10.0.0/16"`. More details [here](https://github.com/terraform-aws-modules/terraform-aws-security-group/issues/16).
```

Comment:
Usage of data-sources is still allowed because it happens before resources are calculated/created. This is a valid alternative to hard-coded values, as you describe it. Am I right? :)

Reply:
Yes, you're totally right! It's not only an alternative, but also a perfectly valid way to pull in values at plan-time that are not treated as calculated. I'll add that in.

Reply:
I ask because I didn't write terraform modules for 7 days and might forget something :) Thanks!